### DISCUSSION TITLE
quarkus:dev setting to exit after execution

### DISCUSSION THREAD

**Discussion by siudeks** [quarkusio/quarkus#29084] [Community]
For development purposes I run quarkus-based application using quarkus:dev inside docker-compose stack. It runs, and next it waits for next actions like *well-known 'Press [space] to restart, [e] to edit command line args (currently ''), [:] for the terminal, [h] for more options>*

How to run application using quarkus:dev, and allows to finish such execution when main thread (*Quarkus Main Thread*) stops working? In my scenario I don't need to restart, test or check any other option, just starting from sources is the goal

**Comment by mkouba** [comment]
I don't think there's an option to run the dev mode in this way.

CC @aloubyansky

**Reply by aloubyansky** [reply]
There is `quarkus:run` feature request for this https://github.com/quarkusio/quarkus/issues/9315 There was a contributor working on it but then it got stuck.
Alternatively, you could probably add a profile to your POM that would build the app and launch it using the `exec` Maven plugin.
